### PR TITLE
samples: add upgrade test

### PR DESCRIPTION
**Pull request by fippo**
adds a sample for upgrading an audio call to an audio-video call.

(I needed something to test Edge and it has shown at least one issue in Chrome too... working around it)

### CODE REVIEW

**Comment by fippo** [comment]
Not sure how the UX of this should be so I went for a minimal approach. Using video first and then adding audio would be a way but this makes testing harder.

File: src/content/peerconnection/upgrade/css/main.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
+/*
+ *  Copyright (c) 2015 The WebRTC project authors. All Rights Reserved.
```
Comment (KaptenJansson) [inline src/content/peerconnection/upgrade/css/main.css]: Please set the copyright to the current year for new files.

**Review by KaptenJansson — APPROVED** [review]
LGTM with a nit.

**Comment by KaptenJansson** [comment]
Merging this regardless of test failure since it's unrelated. Will try to figure out what's causing it.

**Comment by fippo** [comment]
its the lack of addTrack which should be fixed in adapter soon

**Pull request merged**